docs/sources/breaking-changes/breaking-changes-v10-0.md

```diff
@@ -81,7 +81,7 @@ Grafana legacy alerting (dashboard alerts) has been deprecated since Grafana v9.
 
 #### Migration path
 
-The new Grafana Alerting was introduced in Grafana 8 and is a superset of legacy alerting. Learn how to migrate your alerts in the [Upgrade Alerting documentation]({{< relref "../alerting/set-up/migrating-alerts/" >}}).
+The new Grafana Alerting was introduced in Grafana 8 and is a superset of legacy alerting. Learn how to migrate your alerts in the [Upgrade Alerting documentation]({{< relref "./v10.4/alerting/set-up/migrating-alerts/" >}}).
```

Comment:
This `relref` argument doesn't look right to me. It is probably easiest to replace it with the full URL.

```suggestion
The new Grafana Alerting was introduced in Grafana 8 and is a superset of legacy alerting.
Learn how to migrate your alerts in the [Upgrade Alerting documentation](https://grafana.com/docs/grafana/v10.4/alerting/set-up/migrating-alerts/).
```

Reply:
As I mentioned, this link is broken and is now broken on "next": https://grafana.com/docs/grafana/next/breaking-changes/breaking-changes-v10-0/#:~:text=Upgrade%20Alerting%20documentation.